src/index.js

```diff
@@ -147,7 +149,13 @@ export default class ScrubThumbnailsPlugin extends UICorePlugin {
           onLoaded()
         }
       }
-      $img.one("load", onImgLoaded).attr("src", thumb.url)
+      // put image in dom to prevent browser removing it from cache
+      if (cachedImageUrls.indexOf(thumb.url) === -1) {
+        // not been cached
+        cachedImageUrls.push(thumb.url)
```

Comment:
I kept this section if it was needed add the images in the DOM. But in my tests here, I did not need to add because the pictures were cached